.github/workflows/dotnet.yml

```diff
@@ -48,3 +49,43 @@ jobs:
           export DISPLAY=:1.5
           cd lib/PuppeteerSharp.Tests
           dotnet test -f net6.0 -s test.runsettings -c Debug --logger "trx;LogFileName=TestResults.xml"
+
+  deployRelease:
+    if: startsWith( github.ref, 'refs/heads/release-' )
```

Comment:
Does this mean the tags need to begin with `release-` or is this a Github convention?
We could just listen to git tags.

Reply:
Not sure. I used refs/heads/<thing> always to filter branches and used marvinpinto/action-automatic-releases to add tags. I can look that up and report back later.

Are you using tags added by hand to mark releases?

Reply:
I use the Release feature from github that creates a tag.

Reply:
I think you would like to have something that is not filtered to any tag or branch and triggered by you without any automated trigger. Have I got it right? Then it would make sense to pull it out into a new workflow.

Reply:
Ideally. If we can get a new version when I hit "publish release' and a tag is created, that would be awesome.